src/AppBundle/Command/UpdateBundlesListCommand.php

```diff
@@ -0,0 +1,118 @@
+
+        $user = $userService->loadUserByLogin('admin');
+        $permissionResolver->setCurrentUserReference($user);
+
+        $query = new Query();
+        $criterion = new Query\Criterion\ParentLocationId($this->getContainer()->getParameter('bundles.location_id'));
+        $query->filter = $criterion;
+
+        $result = $searchService->findContent($query);
+
+        if ($input->getOption('force')) {
+            $output->writeln('Force option enabled. Updating all packages.');
+        }
+
+        foreach ($result->searchHits as $searchHit) {
+            $currentPackage = $searchHit->valueObject;
+            $package = $packagistServiceProvider->getPackageDetails($currentPackage->getFieldValue('bundle_id'));
+            $output->write($currentPackage->getFieldValue('bundle_id'));
+
+            $packageChecksum = $this->calculateChecksum(array(
+                'updated' => (int) $package['updated']->format('U'),
+                'description' => $package['description'],
+                'downloads' => $package['downloads'],
+                'stars' => $package['stars'],
+                'forks' => $package['forks'],
+            ));
+
+            if (($packageChecksum !== $currentPackage->getFieldValue('checksum')->__toString()) || $input->getOption('force')) {
+                $contentInfo = $contentService->loadContentInfo($searchHit->valueObject->versionInfo->contentInfo->id);
+                $contentDraft = $contentService->createContentDraft($contentInfo);
+
+                $contentUpdateStruct = $contentService->newContentUpdateStruct();
+                $contentUpdateStruct->initialLanguageCode = 'eng-GB';
+                $contentUpdateStruct->setField('updated', (int) $package['updated']->format('U'));
+                $contentUpdateStruct->setField('downloads', $package['downloads']);
+                $contentUpdateStruct->setField('stars', $package['stars']);
+                $contentUpdateStruct->setField('forks', $package['forks']);
+                $contentUpdateStruct->setField('checksum', $packageChecksum);
+
+$xmlText = <<< EOX
+<?xml version='1.0' encoding='utf-8'?>
+<section 
+    xmlns="http://docbook.org/ns/docbook" 
+    xmlns:xlink="http://www.w3.org/1999/xlink" 
+    xmlns:ezxhtml="http://ez.no/xmlns/ezpublish/docbook/xhtml" 
+    xmlns:ezcustom="http://ez.no/xmlns/ezpublish/docbook/custom" 
+    version="5.0-variant ezpublish-1.0">
+<para>{$package['description']}</para>
+</section>
+EOX;
+
+                $contentUpdateStruct->setField('description', $xmlText);
+
+                $contentDraft = $contentService->updateContent($contentDraft->versionInfo, $contentUpdateStruct);
+                $contentService->publishVersion($contentDraft->versionInfo);
+
+                $output->writeln(': Updated');
+            }
+            else {
+                $output->writeln(': OK');
+            }
+        }
+        $output->writeln("All bundles has been successfully updated.");
```

Comment:
It's not all the bundles ;)
typo: `$output->writeln("The bundles have been successfully updated.");`